psd-web/db/migrate/20200402142659_add_is_coronavirus_related_to_investigations.rb

```diff
@@ -0,0 +1,7 @@
+class AddIsCoronavirusRelatedToInvestigations < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured do
+      add_column :investigations, :is_coronavirus_related, :boolean, default: false
```

Comment:
NITPICK:  I do like following the ruby guidelines on method naming https://rubystyle.guide/#bool-methods-prefix
```suggestion
      add_column :investigations, :coronavirus_related, :boolean, default: false
```